From 18dce26c82bbf6f60787fe48a228501fbef0c319 Mon Sep 17 00:00:00 2001
From: Shawn Guo <shawn.guo@freescale.com>
Date: Thu, 12 Jun 2014 15:43:56 +0800
Subject: [PATCH] ENGR00317981: pxp: use DMA_COMPLETE for dma completion
 status
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

DMA_SUCCESS is removed by commit 7db5f7274a0b (dmaengine: remove unused
DMA_SUCCESS).  Let's follow commit 409bff6a0f46 (dmaengine: imx-sdma: use
DMA_COMPLETE for dma completion status) to use DMA_COMPLETE instead.

This is to fix the following build error.

  CC      drivers/dma/pxp/pxp_dma_v2.o
drivers/dma/pxp/pxp_dma_v2.c: In function ‘pxp_tx_status’:
drivers/dma/pxp/pxp_dma_v2.c:1512:9: error: ‘DMA_SUCCESS’ undeclared (first use in this function)

Signed-off-by: Shawn Guo <shawn.guo@freescale.com>
---
 drivers/dma/pxp/pxp_dma_v2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma/pxp/pxp_dma_v2.c b/drivers/dma/pxp/pxp_dma_v2.c
index 0345640..d5b2d8f 100644
--- a/drivers/dma/pxp/pxp_dma_v2.c
+++ b/drivers/dma/pxp/pxp_dma_v2.c
@@ -1509,7 +1509,7 @@ static enum dma_status pxp_tx_status(struct dma_chan *chan,
 		txstate->used = chan->cookie;
 		txstate->residue = 0;
 	}
-	return DMA_SUCCESS;
+	return DMA_COMPLETE;
 }
 
 static int pxp_dma_init(struct pxps *pxp)
-- 
1.8.0

